cli/cmd/inject.go

```diff
@@ -56,10 +57,21 @@ with 'conduit inject'. e.g. curl http://url.to/yml | conduit inject -
 				return err
 			}
 		}
-		return InjectYAML(in, os.Stdout)
+		return runInjectCmd(in, os.Stderr, os.Stdout)
 	},
 }
 
+func runInjectCmd(input io.Reader, errWriter io.Writer, outWriter io.Writer) error {
```

Comment:
This function should not have `error` as its return type. Notice it always returns `nil` so the return value is useless.